app/styles/main.css

```diff
@@ -729,6 +729,8 @@ img.void {
   padding: 2px;
   white-space: nowrap;
   display: inline-block;
+  -webkit-user-select: none;
+  user-select: none;
```

Comment:
I've included this elsewhere.  Expand it to this?

  -webkit-user-select: none;
  -khtml-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;